docs/001_develop/03_client-capabilities/023_custom-components/06_styling-components.mdx

```diff
@@ -65,12 +61,11 @@ import {
 })
 ```
 
-So far we have seen how we can attach styles using the `styles` property in the `@customElement` decorator. But let's dive deeper and explore how we can define styles:
+So far we have seen how we can attach styles using the `styles` property in the `@customElement` decorator. But let's dive deeper and explore how to define styles.
 
-### Defining Styles:
+### Defining styles
 
-In the example below we show how the `css` tagged template helper allows creating and re-using CSS for custom elements.
-These styles are **encapsulated in the Shadow DOM** and attached via the `styles` property in the `@customElement` decorator.
+The example below shows how the `css` tagged template helper enables you to create and re-use CSS for custom elements. These styles are *encapsulated in the Shadow DOM* and attached via the `styles` property in the `@customElement` decorator.
```

Comment:
Is it worth linking to the shadow DOM section down below here as we introduce this concept without explanation?